### ISSUE TITLE
Build for ARM Macs?

### ISSUE THREAD

**Issue by spalger** [bazelbuild/bazel-watcher#493]
I'm trying to develop on a Mac with a fully native toolchain and ibazel is the last tool which doesn't have an ARM build. I've tried running ibazel from brew using rosetta but it spawns a bazel server under rosetta too which causes other problems with the toolchain. I've also tried to build ibazel myself using `bazel build //ibazel` and get errors about `no matching toolchains found for types @io_bazel_rules_go//go:toolchain`...

Is there a chance that someone with more experience could at least put up a build of v0.15.10 for darwin-arm64?

**Comment by mistic** [comment]
Both me and @mrmeku did some experiments around this and ultimately trace the problem to https://github.com/fsnotify/fsevents/issues/50 

Last time we discussed this I was told @gregmagolan knows how to tackle it

**Comment by zoidbergwill** [comment]
Seems like a duplicate of https://github.com/bazelbuild/bazel-watcher/issues/482 ?

Though this issue has more info about how to fix it.

**Comment by zoidbergwill** [comment]
Anyway anyone else can help with this problem, @gregmagolan?

I managed to run `bazel build //ibazel` on my M1 Mac, after updating rules_go, but I'm not sure if there is something else I need to run too.

**Comment by zoidbergwill** [comment]
I also tried to have a look at https://github.com/fsnotify/fsevents/issues/50 , but I think for that we might need to do the different types of builds on machines with different architectures?

**Comment by zoidbergwill** [comment]
Sounds like @mistic figured it out in https://github.com/fsnotify/fsevents/issues/50#issuecomment-1015795295

And hopefully https://github.com/bazelbuild/bazel-watcher/pull/496 will fix it

Thanks @mistic!

**Comment by mistic** [comment]
Now we just need to get the PR I've opened to be able to move forward @zoidbergwill but it is still waiting for reviews

**Comment by achew22** [comment]
Unfortunately all the PRs that claim to fix this do not pass CI. There is no path forward if CI is red, and I don't have an arm mac to do any testing on.

**Comment by mistic** [comment]
@achew22 I dont think that is true, the red CI is being caused by the miss of https://github.com/bazelbuild/bazel-watcher/pull/496/commits/a28f5289b532deca1d8c058fa5e3f6e956bc15a8 which I've reverted as I was not sure about it and was waiting on your review to understand wether or not it make sense. I have arm hardware and I'm available to work with you to move the PR forward as long as you are available to do it...

**Comment by mistic** [comment]
Just reverted the change I mentioned and the CI is now green.
